lib/src/util/react_wrappers.dart

```diff
@@ -71,31 +71,13 @@ bool isDartComponent(/* ReactElement|ReactComponent|Element */ instance) {
   return _getInternal(instance) != null;
 }
 
-@JS('Object.keys')
-external Iterable _objectKeys(object);
-
-/// Returns a Dart Map copy of the JS property key-value pairs in [jsMap].
-Map _dartifyJsMap(jsMap) {
-  return new Map.fromIterable(_objectKeys(jsMap),
-      value: (key) => getProperty(jsMap, key)
-  );
-}
-
 /// Returns the props for a [ReactElement] or composite [ReactComponent] [instance],
 /// shallow-converted to a Dart Map for convenience.
 ///
 /// If `style` is specified in props, then it too is shallow-converted and included
 /// in the returned Map.
 Map getJsProps(/* ReactElement|ReactComponent */ instance) {
```

Comment:
This comment is present in master:
> /// __Deprecated. Use [getProps] instead. Will be removed in 2.0.0.__

Are we planning on removing this in a separate PR, or should it be removed now?